tests/.latest.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
-> .gitignore

and others
```ignore
.bin/
tests/.bin/
tests/.latest.json
subgraph.yaml
deployments/
build/
```

Reply:
Updated  `.gitignore`  and removed unnecessary files.